src/main/java/com/dezorganizacja/bridge/view/graphical/FXApplication.java

```diff
@@ -2,13 +2,20 @@
 
 import com.dezorganizacja.bridge.presenter.MainPresenter;
 import javafx.application.Application;
+import javafx.application.Platform;
 import javafx.fxml.FXMLLoader;
 import javafx.scene.Parent;
 import javafx.scene.Scene;
 import javafx.stage.Stage;
 
-public class FXApplication extends Application {
+import java.io.IOException;
+import java.util.Observable;
+import java.util.Observer;
+
+public class FXApplication extends Application implements Observer {
     private static MainPresenter mainPresenter;
+    private static Stage stage;
+    private static String fxml;
```

Comment:
Dlaczego te zmienne są statyczne? MainPresenter jest statyczny ponieważ to jest taki twardy binding MainPresenteraz w aplikacji będziemy mieli napewno jednego a i tak już to jest brzydkie i może uleć zmianie.